platform/darwin/test/MGLTileSetTests.mm

```diff
@@ -69,10 +69,11 @@ - (void)testTileSetFromTileURLTemplates {
 #if TARGET_OS_IPHONE
     UIColor *redColor = [UIColor redColor];
 #else
-    // CSS uses the sRGB color space. In macOS 10.12 Sierra and below,
-    // -[NSColor redColor] is in the calibrated RGB space and has a slightly
-    // different sRGB value than on iOS and macOS 10.13 High Sierra.
-    NSColor *redColor = [NSColor colorWithSRGBRed:1 green:0 blue:0 alpha:1];
+    // CSS uses the sRGB color space.
+    // AppKit incorrectly uses calibrated RGB when exporting HTML, so input
+    // calibrated RGB to ensure round-tripping.
+    // <rdar://problem/46115233> <http://www.openradar.me/46115233>
+    NSColor *redColor = [NSColor colorWithCalibratedRed:1 green:0 blue:0 alpha:1];
```

Comment:
This is a reprise of mapbox/mapbox-gl-native#13387 that’s now possible because CircleCI is running a newer version of macOS.